### ISSUE TITLE
Demuxlet on Single Nuclei

### ISSUE THREAD

**Issue by achamess** [statgen/demuxlet#27]
Hi,
Has anyone tried Demuxlet on single nuclei using 10x? Does it work?

Also, what are people using for genotyping of human samples?

**Comment by catsargent** [comment]
I would also like to know whether anyone has successfully used Demuxlet on single nuclei data.

Thanks

**Comment by ggruenhagen3** [comment]
Hello, I have used demuxlet on multiple single nuclei datasets using 10x. It works well. It does produce a lot of doublet calls, even on samples that we are very certain are not doublets due to FACs sorting and stringent downstream filtering steps. Seeing some of the posts it seems like others find erroneous doublet calls too, so I just ignore them for the most part.

**Comment by Alkhawaja95** [comment]
@ggruenhagen3 Hi! What do you mean by ignoring doublet assignments? Do you remove these barcodes from your dataset?

**Comment by ggruenhagen3** [comment]
Nope, I keep them. I just ignore the column in demuxlet's output regarding doublets. We had a FACs sorted our samples so we have many layers of evidence that the nuclei were not doublets.
